dags/generate_dags.py

```diff
@@ -18,7 +19,7 @@ def generate_dags():
     client = boto3.client("s3")
     response = client.list_objects_v2(Bucket=bucket, Prefix="collections/")
 
-    for file_ in response["Contents"]:
+    for file_ in response.get("Contents", []):
```

Comment:
If there are no files in the directory, there's no "Contents" in response, which causes the "DAG import Errors (1)"
![image](https://github.com/NASA-IMPACT/veda-data-airflow/assets/7830949/acd6a084-1479-4640-b2d4-4683e6647141)

This fixes that.